packages/compass-schema-validation/src/components/sample-documents.tsx

```diff
@@ -66,47 +73,180 @@ const documentHeadingTextStyles = css({
   color: 'inherit',
 });
 
-export function SampleDocuments() {
+const initialStateContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  height: '228px', // using the same space as the previews
+  gap: spacing[300],
+});
+
+const previewHeaderStyles = css({
+  fontSize: spacing[400],
+  fontWeight: 'bold',
+  color: palette.green.dark2,
+  marginTop: spacing[200],
+});
+
+const previewHeaderDarkModeStyles = css({
+  color: palette.green.light2,
+});
+
+const DocumentGraphic: React.FunctionComponent = () => {
   const darkMode = useDarkMode();
+  const fillColor = useMemo(
+    () => (darkMode ? palette.green.light1 : palette.green.base),
+    [darkMode]
+  );
+  const strokeColor = useMemo(
+    () => (darkMode ? palette.white : palette.black),
+    [darkMode]
+  );
+
   return (
-    <div className={sampleDocumentsSectionStyles}>
-      <Subtitle>
+    <svg
+      width="64"
+      height="58"
+      viewBox="0 0 64 58"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <rect x="0.5" y="1" width="63" height="56" rx="2" stroke={strokeColor} />
+      <line y1="28.5" x2="64" y2="28.5" stroke={strokeColor} />
+      <line
+        x1="8.61035"
+        y1="15.5898"
+        x2="32.9054"
+        y2="15.5898"
+        stroke={strokeColor}
+      />
+      <circle
+        cx="44.9268"
+        cy="16.0898"
+        r="2"
+        fill={fillColor}
+        stroke={strokeColor}
+      />
+      <circle
+        cx="52.8896"
+        cy="16.0898"
+        r="2"
+        fill={fillColor}
+        stroke={strokeColor}
+      />
+    </svg>
+  );
+};
+
+const InitialState: React.FC<{
+  onPreviewClick: () => void;
+}> = ({ onPreviewClick }) => {
+  const darkMode = useDarkMode();
+  return (
+    <div className={initialStateContainerStyles}>
+      <DocumentGraphic />
+      <div
+        className={cx(
+          previewHeaderStyles,
+          darkMode && previewHeaderDarkModeStyles
+        )}
+      >
         <InlineDefinition definition={SAMPLE_DEFINITION}>
-          Sample documents
+          Preview your sample here
```

Comment:
`Preview your sample here` reads a bit funny to me. Maybe `Preview sample documents`?
Not a blocker, fine as is.